test/api/test_dafni_api.py

```diff
@@ -126,3 +127,55 @@ def test_exception_raised_for_failed_call(
         # ASSERT
         with pytest.raises(HTTPError, match="404 client model"):
             dafni_api.dafni_post_request(url, jwt, data)
+
+
+@patch("dafni_cli.api.dafni_api.requests")
+@patch(
```

Comment:
is this necessary? As dafni_put_request doesnt use open, so the data object you use in the tests could be anything

Reply:
Good point. I've removed it.